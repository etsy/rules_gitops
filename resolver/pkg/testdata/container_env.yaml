apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: nginx
  name: nginx-deployment
spec:
  replicas: 3
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
        - env:
            - name: ENVOY_IMAGE_URL
              value: :envoy
            - name: BACKUP_ENVOY_IMAGE_URL
              value: //envoy
          image: nginx
          name: nginx
          ports:
            - containerPort: 80
