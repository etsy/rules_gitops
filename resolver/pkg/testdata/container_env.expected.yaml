apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: nginx
  name: nginx-deployment
spec:
  replicas: 3
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - env:
        - name: ENVOY_IMAGE_URL
          value: docker.io/library/envoy:tag
        - name: BACKUP_ENVOY_IMAGE_URL
          value: docker.io/library/envoy:tag
        image: docker.io/library/nginx:tag
        name: nginx
        ports:
        - containerPort: 80